driver-testsuite/tests/Form/SelectTest.php

```diff
@@ -102,6 +102,43 @@ public function testSetValueMultiSelect()
     }
 
     /**
+     * Set values by Driver
+     *
+     * @dataProvider setValueByOptionTextProvider
+     */
+    public function testSetValueByOptionText($xpath, $value, $expectedResult)
+    {
+        $session = $this->getSession();
+        $session->visit($this->pathTo('/multiselect_form.html'));
+        $page = $session->getPage();
+        $select = $page->find('xpath', $xpath);
+
+        $select->setValue($value);
+        $this->assertEquals($expectedResult, $select->getValue());
+    }
+
+    public function setValueByOptionTextProvider()
```

Comment:
Please rename into `setValueByOptionTextDataProvider` (added `Data` word). Also change name where it's used in test itself.